### PR TITLE
Cg test 13

### PR DESCRIPTION
**Pull request by cgaddis36**
Fixed current_path test validations

### CODE REVIEW

File: spec/features/users/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                                password: "hamburger04",
+                                role: 2
+                              )
+  end
+  describe "when I visit the login page" do
+    it "I am able to submit valid credentials and login as a default user" do
+
+      visit '/login'
+
+      fill_in 'email', with: 'roman@example.com'
```
Comment (adumortier) [inline spec/features/users/login_spec.rb:38]: logs in as a regular user

File: spec/features/users/login_spec.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      click_on 'Log In'
+
+      expect(current_path).to eq('/profile')
+      expect(page).to have_content("You are now logged in!")
+    end
+    it "I am able to submit valid credentials and login as a merchant user" do
+
+      visit '/login'
+
+      fill_in 'email', with: 'alejandro@example.com'
```
Comment (adumortier) [inline spec/features/users/login_spec.rb:50]: logs in as a merchant user

File: spec/features/users/login_spec.rb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      click_on 'Log In'
+
+      expect(current_path).to eq('/merchant/dashboard')
+      expect(page).to have_content("You are now logged in!")
+    end
+    it "I am able to submit valid credentials and login as an admin user" do
+
+      visit '/login'
+
+      fill_in 'email', with: 'kyle@example.com'
```
Comment (adumortier) [inline spec/features/users/login_spec.rb:62]: log in as an admin

File: spec/features/users/login_spec.rb [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+      click_on 'Log In'
+
+      expect(current_path).to eq('/admin/dashboard')
+      expect(page).to have_content("You are now logged in!")
+    end
+    it "I am not able to login with bad credentials" do
+      visit '/login'
+
+      fill_in 'email', with: 'roman37@example.com'
```
Comment (adumortier) [inline spec/features/users/login_spec.rb:73]: log in with credentials that are not in the user table

**Review by adumortier — APPROVED** [review]

**Review by TravisBorgsmiller — APPROVED** [review]

**Comment by TravisBorgsmiller** [comment]
Looks Good!

**Pull request merged**